rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorer.java

```diff
@@ -160,6 +160,11 @@ void setUDPChannel(UDPChannel udpChannel) {
     }
 
     synchronized void handleMessage(DiscoveryEvent event) {
+        logger.debug("co.rsk.net.discovery.PeerExplorer.handleMessage - Handling message:\n" +
```

Comment:
see similar comment above and also check similar log messages below

Reply:
Done